docs/user/user/metrics/data-control-plane/example-scenarios.md

```diff
@@ -25,6 +25,8 @@ Through [Nimbus], we have the ability to sample the population by setting the au
 
 This can be used to slowly roll out instrumentations to the population in order to validate the data we are collecting before measuring the entire population and potentially avoiding costs and  overhead by collecting data that isn’t useful.
 
+> *Note:* this functionality is intended to be used to _sample metrics for a subset of the population_, not for turning metrics on for the entire population. To do that you can just enable the metrics by default by setting `disabled: false` in the metrics.yaml. Then you can "down-sample" if necessary (see Scenario 2 below).
```

Comment:
```suggestion
> *Note:* if planning to use this feature for permanently keeping a data collection on for the whole population, please consider enabling the metrics by default by setting `disabled: false` in the metrics.yaml. Then you can "down-sample" if necessary (see Scenario 2 below).
```

What about this?

Moreover, can you use one of the "important style" boxes we have in other docs?